src/mesh_sandbox/common/__init__.py

```diff
@@ -49,7 +49,7 @@ def strtobool(val: Any) -> Optional[bool]:
 class EnvConfig:
     env: str = field(default="local")
     build_label: str = field(default="latest")
-    auth_mode: str = field(default="no_auth")
+    auth_mode: str = field(default="none")
```

Comment:
Hi Alex ... I think the only question is if auth_mode=none should be the default ..  would it be better to default to "full" .. so it will be a concious choice to ignore the authentication, rather than build a client expecting there to be no auth and get a nasty surprise when you try and move this into production?

Reply:
Yeah agreed that the default being set to full might make sense, I just went with what I thought the intended effect of `no_auth` was